src/widgets/Menu/icons/Logo.tsx

```diff
@@ -82,4 +82,4 @@ const Logo: React.FC<LogoProps> = ({ isDark, ...props }) => {
   );
 };
 
-export default Logo;
+export default React.memo(Logo, (prev, next) => prev.isDark === next.isDark);
```

Comment:
Do you think it is necessary?

Reply:
if the parent re render but the isDark value doesn't change, this component will rerender